docs/v1/openapi.yml

```diff
@@ -812,6 +812,102 @@ paths:
           $ref: "#/components/responses/UnauthorizedError"
         "404":
           $ref: "#/components/responses/DefaultError"
+    put:
+      tags:
+        - "QR code"
+      summary: used for update qrcode by id.
+      security:
+        - bearerAuth: [ ]
+      requestBody:
```

Comment:
parameters:
        - name: id
          in: path
          description: QRcode ID.
          required: true
          schema:
            type: string